Libraries/MiscDrivers/BarcodeDecoder/zbar/src/qrcode/util.h

```diff
@@ -0,0 +1,56 @@
+/*Copyright (C) 2008-2009  Timothy B. Terriberry (tterribe@xiph.org)
+  You can redistribute this library and/or modify it under the terms of the
+   GNU Lesser General Public License as published by the Free Software
+   Foundation; either version 2.1 of the License, or (at your option) any later
+   version.*/
+#if !defined(_qrcode_util_H)
+# define _qrcode_util_H (1)
+
+#define QR_MAXI(_a,_b)      ((_a)-((_a)-(_b)&-((_b)>(_a))))
+#define QR_MINI(_a,_b)      ((_a)+((_b)-(_a)&-((_b)<(_a))))
```

Comment:
I get them in each clean build. I have created a HelloWorld project for MAX78000 and set `LIB_BARCODE_DECODER` to 1.
![zbar](https://user-images.githubusercontent.com/3095300/227445621-6fcadf90-d10d-493c-a818-edb9c068656d.png)

Reply:
This library is ported on MAX32570. 
Do you see any warning on MAX32570?

Reply:
No, building for MAX32570 does not produce any errors. I have just noticed that MAX78000's gcc.mk has `-Wall` set in `CFLAGS` while MAX32570's does not so that may be what is causing the difference in behavior.